packages/ui/src/services/color-config/color-config.ts

```diff
@@ -51,6 +51,10 @@ export const getColor = (prop?: string, defaultColor: string = DEFAULT_COLOR, pr
     return prop
   }
 
+  if (preferVariables && isCSSVariable(prop)) {
```

Comment:
@m0ksem won't be here again a SSR problem because of `isCSSVariable`?

Reply:
`isCSSVariable` uses regex under the hood. CSS variable available in SSR with `renderCSSVariable`. Can't see any problems with SSR.